plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultQueryBuilder.java

```diff
@@ -40,6 +40,7 @@
 import static com.google.common.base.Preconditions.checkState;
 import static com.google.common.base.Verify.verify;
 import static com.google.common.collect.Iterables.getOnlyElement;
+import static io.trino.plugin.jdbc.JdbcMetadataSessionProperties.getDomainCompactionThreshold;
```

Comment:
Commit message

> This reduces number of distinct prepared queries when IN list contains different number of values.

Mention something about "remote" side. Because from Trino perspective, it doesn't matter.